ci/is-container-log-line-present.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -ue
+
+
+main () {
+    if [[ $# -ne 2 ]]; then
+        echo "$0: wrong number of arguments; expected 2, got $#"
+        return 2
+    fi
+    local container_name_substring=$1; shift
+    local log_message=$1; shift
```

Comment:
What's the benefit of `shift` over `$2`?

Reply:
It's easier to add/remove new parameters